desktop/apps/artwork/components/auction/view.coffee

```diff
@@ -23,7 +26,7 @@ module.exports = class ArtworkAuctionView extends Backbone.View
     'change .js-artwork-auction-max-bid': 'setMaxBid'
 
   initialize: ({ @data }) -> #
-    setInterval before(10, @updateBidLabel), 1000
```

Comment:
I wanted to change this pattern, because the way it was before, the timer keeps on firing. It's better to clean up after ourselves by scheduling only the number of calls we want to make, rather than squelch the function calls beyond the ones we want.